easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,409 @@
+name = 'Python'
+version = '2.7.18'
+
+homepage = 'https://python.org/'
+description = """Python is a programming language that lets you work more quickly and integrate your systems
+ more effectively."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://www.python.org/ftp/%(namelower)s/%(version)s/']
+sources = [SOURCE_TGZ]
+checksums = ['da3080e3b488f648a3d7a4560ddee895284c3380b11d6de75edb986526b9a814']
+
+dependencies = [
+    ('binutils', '2.35'),
+    ('bzip2', '1.0.8'),  # required for bz2 package in Python stdlib
+    ('zlib', '1.2.11'),
+    ('libreadline', '8.0'),
+    ('ncurses', '6.2'),
+    ('SQLite', '3.33.0'),
+    ('GMP', '6.2.0'),  # required for pycrypto
+    ('libffi', '3.3'),  # required for cryptography
+    # OS dependency should be preferred if the os version is more recent then this version,
+    # it's nice to have an up to date openssl for security reasons
+    # ('OpenSSL', '1.1.1h'),
+]
+
+osdependencies = [OS_PKG_OPENSSL_DEV]
+
+exts_default_options = {
+    'download_dep_fail': True,
+    'sanity_pip_check': True,
+    'source_urls': [PYPI_SOURCE],
+    'use_pip': True,
+}
+
+# order is important!
+# package versions updated 13 October 2020
+exts_list = [
+    ('setuptools', '44.1.1', {
```

Comment:
We had a comment here to clarify this is the latest version that still supports Python 2, makes sense to retain that?

```python
# setuptools 44.1.1 is the most recent version still supporting Python 2.x

Reply:
Well, IMHO those comments are meaningless. They are technically all the latest version supporting Python 2. 
It doesn't mean that they won't be updated in the future (e.g. pytest), so, you still have to go through the list, and it's starting to apply to something like half the packages.
How about a comment above the list stating this instead of having to add ~15 of these comments (which are stripped out whenever you do a --inject-checksums)

Reply:
OK, one single comment to rule them out makes sense to me